consensus/config.go

```diff
@@ -11,8 +11,8 @@ const (
 	viewChangeDuration time.Duration = 30 * time.Second
 
 	// timeout duration for announce/prepare/commit
-	phaseDuration     time.Duration = 60 * time.Second
-	bootstrapDuration time.Duration = 60 * time.Second
+	phaseDuration     time.Duration = 90 * time.Second
```

Comment:
bootstrapDuration looks good. But phaseDuration is too long which means every time the view change need wait for 90 seconds. It can just be temp work around